test/test-clusters/src/main/java/org/elasticsearch/test/cluster/LogType.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.test.cluster;
+
+public enum LogType {
+    SERVER("elasticsearch.log"),
+    SERVER_JSON("elasticsearch_server.json");
```

Comment:
Don't these all assume a specific cluster name?

Reply:
I was wondering the same thing. Not when I tested it locally. Even with a cluster name of `test-cluster` the log file was just `elasticsearch.log`. What affects this behavior?

Reply:
🤦 I just now noticed we never actually set `cluster.name` in the new framework. Perhaps we should?

Reply:
Ok, I've fixed this. We now set `cluster.name` and resolve the log filename appropriately.